by laws: remove the necessity for the OMC to invite committers and OTC members.

paulidale opened the pull request:
It would be better if these invitations come from the OTC which does the nominations.

This requires an OMC by law vote which will be raised after reviews.

paulidale commented:
Hold awaiting reviews and OMC by law vote.

mspncp commented:
As far as I recall our discussions and infer

@paulidale your pull request title indicates that the bylaw change is intended for committer invitations as well, which was the starting point of our discussion IIRC. However, you are only changing the formulation of the OTC invitations, not the committer invitations as well.  May I suggest the following rewording instead?

```patch
commit 06ce32b95dc15ffc26fd3a3a02515e8a1d578943 (HEAD -> gh-207)
Author: Pauli <paul.dale@oracle.com>
Date:   Fri Nov 6 10:28:55 2020 +1000

    by laws: remove the necessity for the OMC to invite committers and OTC members.
    
    It would be better if these invitations come from the OTC which does the
    nominations.

diff --git a/policies/omc-bylaws.html b/policies/omc-bylaws.html
index c351999..8296f60 100644
--- a/policies/omc-bylaws.html
+++ b/policies/omc-bylaws.html
@@ -49,8 +49,9 @@
           <p>Committers also have a responsibility to review code submissions in
           accordance with OpenSSL project policies and procedures.</p>
 
-          <p>Commit access is granted as a result of a vote by the OMC. It may
-          be withdrawn at any time by a vote of the OMC.</p>
+          <p>Commit access is granted by invitation from the OTC and requires
+          a prior OMC vote of acceptance. It may be withdrawn at any time by
+          a vote of the OMC.</p>
 
           <p>A condition of commit access is that the committer has signed an
           Individual Contributor Licence Agreement (ICLA). If contributions may
@@ -221,10 +222,11 @@
             manner;</li>
           </ul>
 
-          <p>Membership of the OTC is by invitation only from the OMC.
-          OTC members must be committers and hence all rules that apply to committers also apply.
-          OTC members may be OMC members and in which case all rules that apply to OMC members
-          also apply.</p>
+          <p>Membership of the OTC is by invitation from the OTC and requires
+          a prior OMC vote of acceptance. OTC members must be committers and
+          hence all rules that apply to committers also apply.
+          OTC members may be OMC members and in which case all rules that apply
+          to OMC members also apply.</p>
 
           <p>The OTC makes technical decisions on behalf of the project based on
           requirements specified by the OMC. In order to have
```
(this patch replaces your commit, it's not a fixup)

paulidale commented:
@mspncp, your version is better 👍
Feel free to replace mine here or I can.  Or even create a new PR for it.

mspncp commented:
Naah... just drop your HEAD commit and apply the patch (`git am`).

paulidale committed:
by laws: remove the necessity for the OMC to invite committers and OTC members.

It would be better if these invitations come from the OTC which does the
nominations.

paulidale commented:
Okay, done.  I'd not used that git command before and ended up running `patch` directly.

mspncp commented:
> Okay, done. I'd not used that git command before and ended up running `patch` directly.

Whatever suits you best. The advantage of `git am` over `patch` is that it not only applies the change but also commits it using the given commit message. It is optimized for applying entire patch sequences generated by `git format-patch` or `git send-email`.

mspncp commented:
For a single patch, usage is as simple as `git am name-of-patch-file.patch`.

mspncp reviewed (approved):
LGTM, FWIW.

paulidale commented:
This will require an OMC approval.

paulidale commented:
The OMC vote passes five for, none against, no abstentions and two didn't vote.

paulidale commented:
Merged and live.
